### PR TITLE
Lab 1

### PR DESCRIPTION
**Pull request by Sivatharan-STN**

### CODE REVIEW

File: lab-1/Question-2.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <p>Test image 
+        <img src="https://static.toiimg.com/thumb/msid-67586673,width-800,height-600,resizemode-75,imgsize-3918697,pt-32,y_pad-40/67586673.jpg" style="max-width:50px;">alignmets,<br>
+        <img src="https://www.discoverdogs.org.uk/wp-content/uploads/2019/11/dog-hero.jpg" style="max-width:50px;">Test image alignmets,<br>
+        Test image alignmets
+        <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/golden-retriever-royalty-free-image-506756303-1560962726.jpg?crop=0.672xw:1.00xh;0.166xw,0&resize=640:*" style="max-width:50px;"><br>
+    </p>
+    <br>
+    <p>
+        <h4>The pre tag is good for displaying computer code:  </h4>
+        for i = 1 to 10 <br>
```
Comment (kasthurimahadeva) [inline lab-1/Question-2.html]: use pre tag

File: lab-1/Question-3.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</head>
+<body>
+    
+        <h1 style="padding-left: 22%;">World's best scrambles Eggs</h1>
+        <p>This recipe is a very special breakfast: just look at he ingredients to see why it has to be tasted to be believe</p>
+        <h2>Ingredients</h2>
+        <p>The following ingredients make one serving</p>
+
+        <ul>
+            <li type="disc">2 Eggs</li>
```
Comment (kasthurimahadeva) [inline lab-1/Question-3.html]: you can define the "type" within the ul tag

File: lab-1/Question-5.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                Biology<input type="checkbox" id="interest">
+                Chemistry<input type="checkbox" id="interest">
+                Physics<input type="checkbox" id="interest">
+                Computer Science<input type="checkbox" id="interest">
+            </div>
+            
+            <div>
+                <label>Sex:</label><br>
+                <label for="Male">Male</label> 
+                <input type="radio" id="Male"  name="sex" value="male" >
```
Comment (kasthurimahadeva) [inline lab-1/Question-5.html]: "male" should be selected as default

File: lab-1/Question-6.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            <li>Colombo</li>
+            <ol type="1">
+                <li>Borella</li>
+                <li>Kotte</li>
+                <ol type="a" start="b">
+                    <li>Pita Kotte</li>
+                    <li>Ethul Kotte</li>
+                </ol>
+                <li>Nugegoda</li>
+                <li>Maharagama</li>
```
Comment (kasthurimahadeva) [inline lab-1/Question-6.html]: check this

**Review by kasthurimahadeva — COMMENTED** [review]
Q1-5
Q4-10
Q8-10

File: lab-1/Question-2.html (hunk 1 above)
Comment (Sivatharan-STN) [reply lab-1/Question-2.html]: ok miss

File: lab-1/Question-3.html (hunk 2 above)
Comment (Sivatharan-STN) [reply lab-1/Question-3.html]: ok

File: lab-1/Question-5.html (hunk 3 above)
Comment (Sivatharan-STN) [reply lab-1/Question-5.html]: ok

File: lab-1/Question-6.html (hunk 4 above)
Comment (Sivatharan-STN) [reply lab-1/Question-6.html]: done

**Pull request closed without merging**